src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -18,40 +9,339 @@ specified using two files:
       -   [`7t_trt`](https://github.com/bids-standard/bids-examples/tree/master/7t_trt)
       -   [`ds210`](https://github.com/bids-standard/bids-examples/tree/master/ds210)
 
+## General specifications
+
+Continuous (that is, regularly sampled over time at a fixed frequency)
+physiological recordings such as cardiac and respiratory signals,
+asynchronous events corresponding to those signals, and other
+continuous measures (such as parameters of a film or audio stimuli)
+MAY be stored as tabular data in header-less
+[compressed tab-delineated (TSVGZ) files](../glossary.md#tsvgz-extensions).
+TSVGZ are formatted as TSV and MUST be compressed with
+[gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+TSVGZ files MUST be accompanied by a JSON file with the same name as their
+corresponding tabular file but with a `.json` extension.
+Additional [metadata corresponding to TSVGZ files](../common-principles.md#tabular-files)
+MAY be included as prescribed by the common principles.
+For example, indicating units for recorded time series.
+
+??? warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
+
+    In contrast to plain-text TSV files in BIDS, TSVGZ files MUST NOT
+    include a header line.
+    Column names MUST be specified in the JSON file following the
+    [`Columns` metadata](../glossary.md#columns-metadata) specifications.
+    As for all [tabular data structures](../common-principles.md#tabular-files),
+    column names MUST NOT be blank (that is, an empty string),
+    and MUST NOT be duplicated within a single JSON file describing a headerless
+    TSV file.
+
+    TSVGZ are headerless to improve compatibility with existing software
+    (for example, FSL, PNM), and to facilitate the support for other file formats
+    in the future.
+
+Summary template:
+
+<div class="highlight">
+<pre><code><a href="../appendices/entities.html#sub">sub-&lt;label&gt;</a>/
+    [<a href="../appendices/entities.html#ses">ses-&lt;label&gt;</a>/]
+        &lt;<a href="../glossary.html#data_type-common_principles">datatype</a>&gt;/
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>|<a href="../glossary.html#stim-suffixes">stim</a>&gt;<a href="../glossary.html#json-extensions">.json</a>
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>|<a href="../glossary.html#stim-suffixes">stim</a>&gt;<a href="../glossary.html#tsvgz-extensions">.tsv.gz</a>
+</code></pre></div>
```

Comment:
temporary work around because `matches` is not in the glossary yet?

pretty sure the pdf build won't like this. 😉

Reply:
I didn't want it to be temporary -- in addition to matches there is the multiplexing of suffixes, which is also done in some other places. I will see how to fix.

Reply:
@tsalo - should we set up a quick call with @Remi-Gau and figure out how could we update the macros best to cover the two new items this BEP is trying?

- Having wildcard entities such as `<matches>`
- Allow admonitions to decorate macros.

Reply:
@tsalo e.g. - https://github.com/bids-standard/bids-specification/pull/1128#discussion_r1568654523

Reply:
Step 1: getting this approved and merged: https://github.com/bids-standard/bids-specification/pull/1781

Reply:
or even just improve on the this PR so it includes a macro that can deal with this: we could then just use it in the BEP PR